wgpu-types/src/lib.rs

```diff
@@ -828,6 +945,44 @@ pub struct DepthStencilStateDescriptor {
 }
 
 impl DepthStencilStateDescriptor {
+    pub fn new(format: TextureFormat) -> Self {
+        Self {
+            format,
+            depth_write_enabled: false,
+            depth_compare: CompareFunction::Undefined,
```

Comment:
Should `depth_compare` be initialized as `CompareFunction::Undefined`? My worry is that this might fail silently.

Reply:
Sorry, I should have said. We need to accept it as a `new()` parameter instead of having a separate method.